dallinger/utils.py

```diff
@@ -551,6 +609,29 @@ def assemble_experiment_temp_dir(log, config, for_remote=False):
     return dst
 
 
+def copy_file(from_path, to_path):
+    """Actually copy a file from one location to another."""
+    shutil.copyfile(from_path, to_path)
+
+
+def symlink_file(from_path, to_path):
+    """Symbolically link a file from one location to another."""
+    os.symlink(from_path, to_path)
+
+
+def collate_experiment_files(config, experiment_path, destination, copy_func):
```

Comment:
`copy_func` seems like it might be overkill for extensibility, but why not I guess?

Reply:
Could just be a flag, you mean (`method="symlink"` or `method="copy"`)? I'd be OK with that.

Reply:
Yeah, that approach seems more intuitive to me, but it's definitely less flexible. I like your solution and I don't think there's any good reason to change it (especially since this isn't really a public API), it's just a somewhat surprising abstraction.